homeassistant/components/sabnzbd/__init__.py

```diff
@@ -254,7 +108,6 @@ async def async_service_handler(service: ServiceCall) -> None:
 
     async def async_update_sabnzbd(now):
         """Refresh SABnzbd queue data."""
-
         try:
             await sab_api.refresh_data()
             async_dispatcher_send(hass, SIGNAL_SABNZBD_UPDATED, None)
```

Comment:
Side note: A future improvement would be to use our DataUpdateCoordinator to handle refreshing data.

https://developers.home-assistant.io/docs/integration_fetching_data#coordinated-single-api-poll-for-data-for-all-entities